core/internal/messagelog/mocks/mock.go

```diff
@@ -34,23 +34,27 @@ func (m *MockMessageLog) EXPECT() *MockMessageLogMockRecorder {
 }
 
 // Append mocks base method
-func (m *MockMessageLog) Append(arg0 *messages.Message) {
+func (m *MockMessageLog) Append(arg0 *protobuf.Message) {
+	m.ctrl.T.Helper()
```

Comment:
Is this `.Helper()` line relevant to current change? I know this and other changes on mock files are automatically generated by `make generate`, but if so why not included all changes generated by `make generate`? If you have specific reason, please leave some comment on it. Or committing all `make generate` changes is OK, you can do it first at separate patch, then update message interface.

Reply:
I think I just never looked at changes to auto-generated files. But this commit touches too many files, so the changes should be better minimal. I will update the old generated mock files in a separate commit.